src/org/labkey/test/tests/SampleTypeLinkToStudyTest.java

```diff
@@ -0,0 +1,194 @@
+        dgen.addCustomRow(Map.of("name", "second"));
+        dgen.addCustomRow(Map.of("name", "third"));
+        dgen.addCustomRow(Map.of("name", "fourth"));
+        dgen.insertRows(createDefaultConnection(), dgen.getRows());
+
+    }
+
+    @Test
+    public void testLinkToStudy()
+    {
+        goToProjectHome(SAMPLE_TYPE_PROJECT);
+
+        log("Linking sample types one row to study");
+        linkToStudy(SAMPLE_TYPE1, 2, Arrays.asList("P1", "P2"), Arrays.asList("1", "2"));
+
+        log("Verifying the linked sample type in study");
+        goToProjectHome(LINKED_STUDY_PROJECT);
+        clickAndWait(Locator.linkWithText(SAMPLE_TYPE1));
+        DataRegionTable table = new DataRegionTable("Dataset", getDriver());
+
+        checker().verifyEquals("Incorrect number of rows linked", 2, table.getDataRowCount());
+        checker().verifyEquals("Incorrect Participant ID's", Arrays.asList("P1", "P2"), table.getColumnDataAsText("ParticipantId"));
+
+        log("Verifying log entries");
+        goToProjectHome(SAMPLE_TYPE_PROJECT);
+        clickAndWait(Locator.linkWithText(SAMPLE_TYPE1));
+        String expectedComment = "2 row(s) were linked to a study from the sample type: " + SAMPLE_TYPE1;
+        verifyLinkToHistory(expectedComment);
+        verifyAuditLogEvents(expectedComment);
+    }
+
+    @Test
+    public void testDatasetRecall()
+    {
+        goToProjectHome(SAMPLE_TYPE_PROJECT);
+        linkToStudy(SAMPLE_TYPE2, 1, Arrays.asList("P3"), Arrays.asList("3"));
+
+        goToProjectHome(LINKED_STUDY_PROJECT);
+        clickAndWait(Locator.linkWithText(SAMPLE_TYPE2));
+        DataRegionTable table = new DataRegionTable("Dataset", getDriver());
+        table.checkCheckbox(0);
+        table.clickHeaderButton("Recall");
+        acceptAlert();
+        checker().verifyEquals("Dataset row not recalled", 0, table.getDataRowCount());
+
+        log("Verifying log entries");
+        goToProjectHome(SAMPLE_TYPE_PROJECT);
+        clickAndWait(Locator.linkWithText(SAMPLE_TYPE2));
+        String expectedComment = "1 row(s) were recalled from a study to the sample type: " + SAMPLE_TYPE2;
+        verifyLinkToHistory(expectedComment);
+        verifyAuditLogEvents(expectedComment);
+    }
+
+    private void linkToStudy(String sampleName, int numOfRowsToBeLinked, List<String> participantID, List<String> visitId)
+    {
+        clickAndWait(Locator.linkWithText(sampleName));
+        DataRegionTable samplesTable = DataRegionTable.DataRegion(getDriver()).withName("Material").waitFor();
+        for (int i = 0; i < numOfRowsToBeLinked; i++)
+        {
+            samplesTable.checkCheckbox(i);
+        }
+
+        samplesTable.clickHeaderButtonAndWait("Link to Study");
+
+        log("Link to study: Choose target");
+        selectOptionByText(Locator.id("targetStudy"), "/" + LINKED_STUDY_PROJECT + " (" + LINKED_STUDY_PROJECT + " Study)");
+        clickButton("Next");
+
+        log("Lick to Study: Verify Results");
```

Comment:
I think you want to "Click to study". :)

Reply:
Yes offcourse